src/quote/Quote.ts

```diff
@@ -77,6 +104,15 @@ export function createAttesterSignature(
   }
 }
 
+/**
+ * Creates a [[Quote]] object signed by the given [[Identity]].
+ *
+ * @param quoteInput A [[Quote]] object.
+ * @param identity [[Identity]] to add the public address of the Attester.
```

Comment:
Does it really add the public address? Isn't the identity solely used for signing?

Reply:
You are right, we are only signing with it. 

I have updated it accordingly.